app/renderer/components/bookmarksToolbar.js

```diff
@@ -47,6 +48,21 @@ class BookmarkToolbarButton extends ImmutableComponent {
     }
   }
 
+  onMouseOver (e) {
+    // Behavior when a bookmarks toolbar folder has its list expanded
+    if (this.props.selectedFolderId) {
```

Comment:
@bridiver:
1. I can update the code to do this.  It would be a larger change and would put "business logic" into the WindowStore.  For example, if the selection changes from null or -1 to a valid number, the WindowStore would know it needs to create a ContextMenu renderer control, which I guess it would handle by doing another windowAction?  Maybe we can talk this out more

2. This particular method (`showBookmarkFolderMenu`) doesn't use `electron.remote`. However, there are methods in that file (`contextMenus.js`) which do (specifically for creating / showing popup menu templates)